app/src/main/java/hous/release/android/presentation/settings/SettingsActivity.kt

```diff
@@ -0,0 +1,93 @@
+import android.net.Uri
+import android.os.Build
+import android.os.Bundle
+import hous.release.android.BuildConfig
+import hous.release.android.R
+import hous.release.android.databinding.ActivitySettingsBinding
+import hous.release.android.util.binding.BindingActivity
+
+class SettingsActivity : BindingActivity<ActivitySettingsBinding>(R.layout.activity_settings) {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        initSettingsMode()
+        initBackClickListener()
+        initNotiSettingClickListener()
+        initInfoClickListener()
+        initFeedbackClickListener()
+    }
+
+    private fun initSettingsMode() {
+        binding.hasRoom = intent.getBooleanExtra(HAS_ROOM, true)
+    }
+
+    private fun initBackClickListener() {
+        binding.btnSettingsBack.setOnClickListener { finish() }
+    }
+
+    private fun initNotiSettingClickListener() {
+        binding.btnSettingsNotification.setOnClickListener {
+            startActivity(Intent(this, NotificationSettingsActivity::class.java))
+        }
+    }
+
+    private fun initInfoClickListener() {
+        binding.btnSettingsInfo.setOnClickListener {
+            startActivity(
+                Intent(Intent.ACTION_VIEW, Uri.parse(getString(R.string.settings_info_link)))
+            )
+        }
+    }
+
+    private fun initFeedbackClickListener() {
+        binding.btnSettingsFeedback.setOnClickListener {
+            startActivity(
+                Intent(Intent.ACTION_SEND).apply {
+                    type = "plain/text"
+                    putExtra(
+                        Intent.EXTRA_EMAIL,
+                        arrayOf(getString(R.string.settings_feedback_email))
+                    )
+                    putExtra(Intent.EXTRA_SUBJECT, getString(R.string.settings_feedback_subject))
+                    putExtra(
+                        Intent.EXTRA_TEXT,
+                        getString(
+                            R.string.settings_feedback_content,
+                            getDeviceName(),
+                            Build.VERSION.SDK_INT.toString(),
+                            BuildConfig.VERSION_NAME
+                        )
+                    )
+                }
+            )
+        }
+    }
+
+    private fun getDeviceName(): String {
+        val manufacturer = Build.MANUFACTURER
+        val model = Build.MODEL
+        return if (model.startsWith(manufacturer)) {
+            capitalize(model)
+        } else {
+            "${capitalize(manufacturer)} $model"
+        }
+    }
+
+    private fun capitalize(str: String?): String {
+        if (str == null || str.isEmpty()) {
+            return getString(R.string.settings_feedback_empty_string)
+        }
+        val first = str[0]
+        return if (Character.isUpperCase(first)) {
+            str
+        } else {
+            Character.toUpperCase(first).toString() + str.substring(1)
+        }
+    }
```

Comment:
```suggestion
private fun capitalize(str: String?): String {
    if (str.isNullOrEmpty()) return getString(R.string.settings_feedback_empty_string)
    return str.replace(str[0], str[0].uppercaseChar())
}
```
String의 첫번 째 문자를 대문자로 바꿔서 보내야하는 걸까요??  
위와 같이 simple하게 나타낼 수 있을 것 같습니당 ㅎ ㅎ

Reply:
첫번째 자리가 이미 대문자면 안바꾸려고 저렇게 한거긴 한데 이게 더 깔꼼해본이네요!!!ㅎㅎ

Reply:
![image](https://user-images.githubusercontent.com/87055456/199776267-3290f15e-cbe6-4251-9601-af3ef3531e5a.png)  
- 출력결과
![image](https://user-images.githubusercontent.com/87055456/199776328-7362df49-5348-49ba-8483-f7a98b1a888f.png)    

string의 0번째 character만 바꾸는거라 괜찮아유!!